.github/workflows/CI.yml

```diff
@@ -24,21 +24,17 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          environment-file: environment.yml
-          python-version: 3.7
-          activate-environment: rms_env
-      - name: Fix PyCall linking
-        run: |
-            julia -e 'ENV["PYTHON"]=ENV["CONDA"] * "/envs/rms_env/bin/python"; using Pkg; Pkg.add("PyCall"); Pkg.build("PyCall")'
       - uses: julia-actions/julia-buildpkg@v1
       - name: Build
         run: |
-            julia -e 'using Pkg; Pkg.develop(PackageSpec(path="../ReactionMechanismSimulator.jl")); Pkg.build("ReactionMechanismSimulator");'
+            current_path=${{ github.workspace }}
+            export JULIA_CONDAPKG_ENV="$current_path/rms_env"
+            julia -e 'using Pkg; Pkg.develop(Pkg.PackageSpec(path="../ReactionMechanismSimulator.jl/")); Pkg.build("ReactionMechanismSimulator");'
       - name: Run tests
         run: |
-            julia -e 'using Pkg; Pkg.test("ReactionMechanismSimulator";coverage=true)'
+            current_path=${{ github.workspace }}
+            export JULIA_CONDAPKG_ENV="$current_path/rms_env"
```

Comment:
Is there a reason we need to repeat this everytime?

Reply:
I don't think we need to, but I think it doesn't hurt to be explicit